docker-deploy.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash -ex
```

Comment:
Is that better than containing all docker-related things in here? I'd prefer to keep it in one place.